src/css/profile/mobile/common/toast.less

```diff
@@ -85,21 +71,6 @@
 	}
 }
 
-@-webkit-keyframes ui-smallpopup-show {
```

Comment:
Is it a keyframe that doesn't even need a webkit?

Reply:
As far as I know, this is added during build by autoprefixer.

Reply:
ok. I got it.